images/scripts/get-image-digest.sh

```diff
@@ -12,4 +12,16 @@ if [ "$#" -ne 1 ] ; then
   exit 1
 fi
 
-docker buildx imagetools inspect "${1}" 2>/dev/null | grep Digest | awk '{ print $2 }' || true
+if [ "$(uname)" != "Linux" ]; then
+  # shellcheck disable=SC2120
+  function sha256sum() { openssl sha256 "$@" | awk '{print $2}'; }
```

Comment:
```suggestion
  function sha256sum() { openssl sha256; }
```

This doesn't work on mac OS. I verified that the suggested change works.